### ISSUE TITLE
Unable to declare global variable as alias to existing type

### ISSUE THREAD

**Issue by OliverJAsh** [Microsoft/TypeScript#10305]
Hi, I have a small issue when trying to declare a global variable as an alias to an existing type.

TypeScript 1.8.7

Given the following declaration file:

``` ts
// typings.d.ts
import { Assert } from 'chai/lib/Assert';

declare const assert: Assert;
declare const helper: any;
```

If I reference this declaration file in my project, both `assert` and `helper` cannot be found. I expect them both to be found.

``` ts
// main.ts
assert; // unexpected error
helper; // unexpected error
```

If I comment out lines 1 and 3, helper can be found as expected.

``` ts
// main.ts
assert; // error as expected
helper;
```

What is wrong with my declaration file and why do I not get any feedback in my declaration file about this?

**Comment by RyanCavanaugh** [comment]
A file that `import`s or `export`s things is a *module* and only contributes names to its own namespace, not the global namespace. You can put these things in a `declare global {` block if you want to put them into the global namespace instead.

**Comment by OliverJAsh** [comment]
Fantastic, that makes perfect sense. Thanks for the speedy reply. Not sure whether to post these questions here or on IRC, what would you advise?

Is this `declare global` documented anywhere?

**Comment by RyanCavanaugh** [comment]
StackOverflow would be preferred.

I think we're behind on docs in this area.

**Comment by OliverJAsh** [comment]
``` ts
import { Assert } from 'chai/lib/Assert';

declare global {
	const assert: Assert;
	const helper: any;
}
```

I get the error:

```
server/manual-typings.d.ts(4,8): error TS2665: Module augmentation cannot introduce new names in the top level scope.
```

**Comment by RyanCavanaugh** [comment]
That error will be removed after TS 1.8. If you're able to try the 2.0 beta or the latest bits (`npm install typescript@next`) you can see how it works.
